packages/perseus/src/widgets/__tests__/interactive-graph.test.ts

```diff
@@ -142,56 +141,19 @@ describe("mafs graphs", () => {
         flags: {mafs: graphTypeFlags},
     };
 
-    describe("locked layer", () => {
-        it("should render locked points", async () => {
-            // Arrange
-            const {container} = renderQuestion(
-                segmentWithLockedPointsQuestion,
-                apiOptions,
-            );
-
-            // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
-            const points = container.querySelectorAll(
-                // Filter out the interactive points' circles
-                "circle:not([class*='movable-point'])",
-            );
-
-            // Act
-
-            // Assert
-            expect(points).toHaveLength(2);
-        });
-
-        test("should render locked points with styles", async () => {
-            // Arrange
-            const {container} = renderQuestion(
-                segmentWithLockedPointsQuestion,
-                apiOptions,
-            );
-
-            // Act
-            // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
-            const points = container.querySelectorAll(
-                "circle:not([class*='movable-point'])",
-            );
-
-            // Assert
-            expect(points[0]).toHaveStyle({fill: color.red, stroke: color.red});
-            expect(points[1]).toHaveStyle({fill: color.red, stroke: color.red});
-        });
-    });
-
-    describe("segment", () => {
+    describe.each([
+        segmentQuestionDefaultCorrect,
+        linearQuestionWithDefaultCorrect,
+        linearSystemQuestionWithDefaultCorrect,
+        rayQuestionWithDefaultCorrect,
+    ])("segment", (question) => {
```

Comment:
Since all of the sets of tests are basically identical, we can just loop through them this way